integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/ClusterUtils.java

```diff
@@ -67,28 +67,28 @@ public static ClusterResource createClusterResource(String clusterResName,
    */
   public static void createClusterAndVerify(ClusterResource cluster) {
     LoggingFacade logger = getLogger();
-    String clusterName = cluster.getSpec().getClusterName();
+    String clusterResourceName = cluster.getClusterResourceName();
     String namespace = cluster.getNamespace();
     // create the cluster CR
     assertNotNull(cluster, "cluster is null");
     assertNotNull(cluster.getSpec(), "cluster spec is null");
-    assertNotNull(clusterName, "clusterName is null");
+    assertNotNull(clusterResourceName, "clusterResourceName is null");
 
     logger.info("Creating cluster custom resource for clusterName {0} in namespace {1}",
-        clusterName, namespace);
+        clusterResourceName, namespace);
     assertTrue(assertDoesNotThrow(() -> createClusterCustomResource(cluster),
             String.format("Create cluster custom resource failed with ApiException for %s in namespace %s",
-                clusterName, namespace)),
+                clusterResourceName, namespace)),
         String.format("Create cluster custom resource failed with ApiException for %s in namespace %s",
-            clusterName, namespace));
+            clusterResourceName, namespace));
```

Comment:
If createClusterCustomResource returns false then there is no ApiException was thrown then this assert message will be misleading.